src/overview/pricing/sponsored.md

```diff
@@ -0,0 +1,39 @@
+# Sponsored Community Sites
+
+Platform.sh provides sponsored hosting for Free Software projects and tech community events and organizations as part of our effort to support the Free Software community.
+
+## Eligibility
+
+* A non-trivial software project released under a Free and Open Source license (*GPL, BSD, MIT, Apache, etc.)
+* A community-oriented conference or similar event run either by the community or a related non-profit corporation.  For-profit events and organizations are not eligible.
+* An offline IT community organization, such as a local user group for a Free and Open Source project.
+* An online non-profit community project that supports one of the above.
```

Comment:
@Crell 
Can we include some social project such as https://glocal.org.br/aceleradora/